lib/validates_zipcode/formatter.rb

```diff
@@ -13,6 +13,13 @@ class Formatter
       PL: ->(z) { z.scan(/\d/).insert(2, '-').join },
       SK: :CZ,
       UK: :GB,
+      MT: :GB,
```

Comment:
We may need to add bespoke (or remove this) formatting for Malta as it's a slightly different format.